src/lib/singelton/loggerService.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger, ETransportType, ELoglevel } from "../logger/index";
+
+export class LoggerSing extends Logger
+{
+    private static _instance: LoggerSing;
+
+    public constructor(loglevel)
+    {
+        super({
+            baseComment: "ser-ext-ondemand",
+            loglvl: loglevel,
+            transports: [
+                {
+                    showBaseComment: true,
+                    showDate: true,
+                    showLoglevel: true,
+                    type: ETransportType.console
+                }
+            ]
+        })
+        // new Logger();
+    }
+
+    public static get Instance()
+    {
+        return this._instance;
```

Comment:
this one should create now an instance like this one

```ts
if (!this._instance) {
    this._instance = new LoggerSing();
}
return this._instance;
```

Reply:
removed Singolton, fixed with https://github.com/senseexcel/ser-ext-ondemand/commit/39ff29d840b6de7135be57615e679bf72f18be35